src/server/feedback/feedbackroutes.js

```diff
@@ -83,6 +87,16 @@ const routes = app => {
     });
 };
 
+const reportMetrics = requestBody => {
+    const parsed = JSON.parse(requestBody.txt);
+    const behandlingId = requestBody.id;
+    if (parsed.uenigheter.length == 0) {
+        counter.agree(behandlingId);
+    } else {
+        parsed.uenigheter.forEach(uenighet => counter.disagree(behandlingId, uenighet.label));
```

Comment:
Poeng å telle både antall uenigheter og antall innsendinger? Jeg vet ikke hvor balansen går mellom å logge for mye og skape støy, eller logge mer enn nødvendig for fremtidige hensyns skyld 😛

Reply:
Tanken er å kunne gruppere i Grafana. Antall feil per innsending, hvordan er uenigheter fordelt osv.